### PR TITLE
NIAD-1621: GPCC Release 0.0.6

### PR DESCRIPTION
**Pull request by ElliotB-96**
GPC 1.6.0 migrate endpoints added

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-1296829
+- https://snyk.io/vuln/SNYK-JAVA-IONETTY-1042268
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316638
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316639
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316640
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316641
+- https://snyk.io/vuln/SNYK-JAVA-ORGGLASSFISH-1297098
+
+### Added
+add url replacement for migrate structure doc refs (#50)
```
Comment (Broom94) [inline CHANGELOG.MD:27]: add ticket number here

File: CHANGELOG.MD [hunk 2]
```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.6] - 2021-07-19
+
```
Comment (Broom94) [inline CHANGELOG.MD:10]: could write quick summary here saying gpc 1.6.0 endpoints added or something

File: CHANGELOG.MD [hunk 3]
```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- https://snyk.io/vuln/SNYK-JAVA-ORGECLIPSEJETTY-1313686
+- https://snyk.io/vuln/SNYK-JAVA-ORGECLIPSEJETTY-1300835
+- https://snyk.io/vuln/SNYK-JAVA-ORGECLIPSEJETTY-1319666
+- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-1296829
+- https://snyk.io/vuln/SNYK-JAVA-IONETTY-1042268
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316638
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316639
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316640
+- https://snyk.io/vuln/SNYK-JAVA-ORGAPACHECOMMONS-1316641
+- https://snyk.io/vuln/SNYK-JAVA-ORGGLASSFISH-1297098
```
Comment (Broom94) [inline CHANGELOG.MD:24]: are these dependency related / can they be updated and addressed?

Comment (ElliotB-96) [reply CHANGELOG.MD:24]: Tried updating the dependencies and they weren't being applied

File: CHANGELOG.MD [hunk 4]
```diff
@@ -6,6 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Added
+NIAD-1418: add url replacement for migrate structure doc refs (#50)
+NIAD-1508: add a route to GPC's 'migrate document' endpoint (#48)
+NIAD-1417: Migrate Patient Record (#47)
+NIAD-1518: support multiple orgs (#46)
+NIAD-1533: GPCC mocks (#45)
+NIAD-1517: Refactoring required to enable multiple GPC providers (#43)
+
+
```
Comment (Broom94) [inline CHANGELOG.MD:35]: remove extra line here

**Review by Broom94 — APPROVED** [review]

File: release/DOCKERHUB.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ you are testing to ensure compatibility with configurations and scripts.
 
 ```bash
 git pull
-git checkout 0.0.5
+git checkout 0.0.6
```
Comment (Broom94) [inline release/DOCKERHUB.md:26]: change to 0.1.5

File: release/version.sh [hunk 6]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-export RELEASE_VERSION=0.0.5
+export RELEASE_VERSION=0.0.6
```
Comment (Broom94) [inline release/version.sh:3]: change to 0.1.5

File: CHANGELOG.MD [hunk 7]
```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.6] - 2021-07-19
```
Comment (Broom94) [inline CHANGELOG.MD:9]: change to 0.1.5

**Review by Broom94 — APPROVED** [review]

**Review by dawidskainos — APPROVED** [review]
